docs/specification/attributes.md

```diff
@@ -50,7 +51,7 @@ Field name   | Value type | Default | Description
 `value`      | [Value](../value-types) | Req. | Any `Value` type
 `name`       | String     | Req. | The name of the attribute, which is used to identify it within a Data Object
 `notes`      | String     | None | Some free-form notes about the attribute.
-`origin`     | `measured`, `predicted`, `specified`, `computed`, `unknown` | `unknown` | The origin of the attribute
+`origin`     | `measured`, `predicted`, `summary`, `specified`, `computed`, `unknown` | `unknown` | The origin of the attribute
```

Comment:
Might be worth noting that the different versions of this list have different orders.

Reply:
It's a set, so order is 🤷